src/main/java/edu/wpi/grip/Main.java

```diff
@@ -1,5 +1,33 @@
 package edu.wpi.grip;
 
-public class Main {
-    public static void main(String[] args) {}
+import javafx.application.Application;
+import javafx.beans.value.ObservableValue;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Cursor;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Screen;
+import javafx.stage.Stage;
+
+public class Main extends Application {
```

Comment:
How are we supposed to launch this application in headless mode now?
Shouldn't the application entry point be in `src/main/java/edu/wpi/grip/ui`?